libs/sr/db/src/Database/DataManagers/LightConeData.ts

```diff
@@ -250,3 +239,18 @@ export class LightConeDataManager extends SroDataManager<
     return { duplicated, upgraded }
   }
 }
+
+export function validateLightCone(obj: unknown = {}): ILightCone | undefined {
+  if (typeof obj !== 'object') return undefined
+  const { key, level: rawLevel, ascension: rawAscension } = obj as ILightCone
+  let { superimpose, location, lock } = obj as ILightCone
+
+  if (!allLightConeKeys.includes(key)) return undefined
+  if (rawLevel > lightConeMaxLevel) return undefined
+  const { level, ascension } = validateLevelAsc(rawLevel, rawAscension)
+  if (typeof superimpose !== 'number' || superimpose < 1 || superimpose > 5)
+    superimpose = 1
+  if (location && !allCharacterLocationKeys.includes(location)) location = ''
```

Comment:
```suggestion
  if (!location || !allCharacterLocationKeys.includes(location)) location = ''
```
This will fix the location not being autopopulated